webapp/runtime/src/main/java/org/wildfly/swarm/topology/webapp/runtime/TopologyProxyService.java

```diff
@@ -97,8 +98,8 @@ public void onChange(Topology topology) {
     }
 
     private void updateProxyHosts(String serviceName, List<Topology.Entry> entries) {
-        ProxyHandler proxyHandler = proxyHandlerMap.get(serviceName).getValue();
-        LoadBalancingProxyClient proxyClient = (LoadBalancingProxyClient) proxyHandler.getProxyClient();
+        HttpHandler proxyHandler = proxyHandlerMap.get(serviceName).getValue();
```

Comment:
Yes this is not very nice.That was my first workaround solution. Need to think about something better...